be/src/common/config.h

```diff
@@ -615,6 +615,8 @@ CONF_mInt32(tablet_meta_checkpoint_min_interval_secs, "600");
 CONF_Int64(brpc_max_body_size, "2147483648");
 // Max unwritten bytes in each socket, if the limit is reached, Socket.Write fails with EOVERCROWDED.
 CONF_Int64(brpc_socket_max_unwritten_bytes, "1073741824");
+// brpc connection types, "single", "pooled", "short".
+CONF_String(brpc_connection_type, "single");
```

Comment:
needs to check the validness of the input setting. We need to work well even user give a wrong config

Reply:
fixed